src/espIdf/apptrace/tree/appTraceArchiveTreeDataProvider.ts

```diff
@@ -51,7 +51,7 @@ export class AppTraceArchiveTreeDataProvider
     return this.appTraceArchives;
   }
   public refresh() {
-    this.OnDidChangeTreeData.fire();
+    this.OnDidChangeTreeData.fire(null);
```

Comment:
@brianignacio5 I tried (latest changes) without passing `null` it compiles fine and run as expected.
What is the error you got here ?